keys.go

```diff
@@ -178,9 +193,18 @@ func (c *Client) GetKeys(ctx context.Context, limit int, offset int) (*Keys, err
 
 // GetKey retrieves a key by ID.
 func (c *Client) GetKey(ctx context.Context, id string) (*Key, error) {
+	return c.getKey(ctx, id, "keys/%s")
+}
+
+// GetKeyMetadata retrieves the metadata of a Key.
+func (c *Client) GetKeyMetadata(ctx context.Context, id string) (*Key, error) {
```

Comment:
Since we are returning a Key ref here, is the expectation that the Payload attribute will be empty then since this is just metadata?

Probably worth adding a test for this and maybe a quick mention of that in the function comment to indicate how its different.

Reply:
Done.